package.json

```diff
@@ -1,12 +1,14 @@
 {
   "name": "variable-type",
-  "version": "0.2.1",
+  "version": "1.0.0",
```

Comment:
0.3.0 is the next break change version.

Reply:
Agreed, this lib is pretty stable though, don't you think? Worthy of a 1.0 IMO. I've been using it for years and it does what I expect it to which is replicate the prop-types api effectively for use server-side. I don't have a strong opinion either way honestly but I think for some team members, seeing a pre-1.0 version number can cause concern.